docs/HTTP3.md

```diff
@@ -0,0 +1,85 @@
+# HTTP3 (and QUIC)
+
+## Resources
+
+[HTTP/3 Explained](https://daniel.haxx.se/http3-explained/) - the online free
+book describing the protocols involved.
+
+[QUIC implementation](https://github.com/curl/curl/wiki/QUIC-implementation) -
+the wiki page describing the plan for how to support QUIC and HTTP/3 in curl
+and libcurl.
+
+[quicwg.org](https://quicwg.org/) - home of the official protocol drafts
+
+## QUIC libraries
+
+QUIC libraries we're experiementing with:
+
+[ngtcp2](https://github.com/ngtcp2/ngtcp2)
+
+[quiche](https://github.com/ghedo/quiche)
+
+## Experimental!
+
+HTTP/3 and QUIC support in curl is not yet working and this is early days.
+Consider all QUIC and HTTP/3 code to be **EXPERIMENTAL** until further notice.
+
+curl does not have HTTP/3 support (yet).
+
+# ngtcp2 version
+
+The bleeding edge QUIC work is done in the dedicated
+[ngtcp2](https://github.com/curl/curl/tree/ngtcp2) branch, but the plan is to
```

Comment:
This should be https://github.com/curl/curl/tree/QUIC, right?

Reply:
Yes, thank you. I changed branch but forgot to update the docs!